RELEASE_NOTES.md

```diff
@@ -0,0 +1,69 @@
+# 3.4 
+
+The top four improvements in Dafny 3.4 are:
+- For certain classes of changes, prevent unexpected changes in verification behavior when changing a Dafny program.
+- Add command line options to assist in debugging verification performance.
+- Critical fixes to the IDE and greatly improved responsiveness of non-verification IDE features. 
+- The C# back-end supports traits as type parameters on datatypes.
+
+### Verification
+- fix: Resolve unsoundness issue (https://github.com/dafny-lang/dafny/issues/1619).
+- fix: Don't silently succeed if the solver crashes (https://github.com/boogie-org/boogie/pull/488).
+- feat: Prevent changes in the verification behavior of a proof, when any of these types of changes are made to Dafny user code:
+  - Changes logically unrelated to the proof being verified
+  - Changes to the name of any declaration
+  - Changes to the order of top-level declarations
+- feat: Assist in debugging the verification performance of a proof by adding the `/vcsSplitOnEveryAssert` CLI option and `{:vcs_split_on_every_assert}` attribute, and report the outcome and duration of splits when they occur in `/verificationLogger:trx` content.
```

Comment:
```suggestion
- feat: Assist in debugging the verification performance of a proof by adding the `/vcsSplitOnEveryAssert` CLI option and `{:vcs_split_on_every_assert}` attribute (see https://github.com/boogie-org/boogie/issues/465), and report the outcome and duration of splits when they occur in `/verificationLogger:trx` content.
```

Seems fair to note that the first part is actually a Boogie feature (especially since we'll want to make it a much more polished Dafny feature soon: https://github.com/dafny-lang/dafny/issues/1754)